truffle-config.js

```diff
@@ -16,6 +16,24 @@ module.exports = {
   //    port: 7545,
   //    network_id: "*"
   //  }
-  //}
+  //},
+
   //
+  db: {
+    // Truffle-DB is disabled by default. 
+    enabled: false,
+    host: "127.0.0.1",
+
+    // You can also use a local db storage adapter by uncommenting this section
+    //
+    adapter: {
+      name: "sqlite",
+      settings: {
+        // You may also choose a differnt directory name
+        // Note. it will alwasy be relative to Project Root
+        // (the same folder where this file is located.
+        directory: ".truffle-db"  
```

Comment:
Since we use `.db` at the user-level for this, what do you think of making it `.db` here as well?

Reply:
I want it to stand out from `.db` so there's less opportunity for confusion. if someone refers to not `.db` we'll be reminded to look at truffle config

Reply:
That being said, I'm ok to change it to `.db` if you want.

Reply:
I don't have a strong feeling about it, honestly. I like the consistency of naming I'm suggesting but I see the usefulness of your approach. Maybe @gnidan has feelings about it. Otherwise, I'm fine with whatever you choose here!

Reply:
I prefer `.db` for consistency

Reply:
outvoted! `.db` it is